bin/process-extensions-scss.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const sass = require( 'node-sass' );
+
+function getExtensionsWithScss() {
+	const extensionsDirectory = path.resolve( __dirname, '..', 'client', 'extensions' );
+	const extensionsNames = fs.readdirSync( extensionsDirectory ).filter( filename => {
+		// Return if this is a file (has extension)
+		if ( filename.indexOf( '.' ) !== -1 ) {
```

Comment:
Would it be more robust to use isDirectory? https://stackoverflow.com/questions/15630770/node-js-check-if-path-is-file-or-directory